src/binding.c

```diff
@@ -28,39 +36,95 @@ NAPI_METHOD(cpuRdtsc)
   NAPI_ARGV_INT32(n_samples, 1);
   NAPI_ARGV_INT32(interval, 2);
 
+  n_samples = CLAMP(n_samples, MAX_SAMPLE_SIZE, 0);
+  interval = CLAMP(interval, MAX_INTERVAL_MILLISECONDS, 0);
```

Comment:
should we CLAMP silently or throw an exception?

Reply:
I can throw if you want to, I guess that it's all about setting clear expectations I guess